linux/net/rina/pci.c

```diff
@@ -490,3 +490,19 @@ EXPORT_SYMBOL(pci_control_my_left_wind_edge);
 seq_num_t pci_control_last_seq_num_rcvd(const struct pci * pci)
 { return pci ? pci->control.last_ctrl_seq_num_rcvd : 0; }
 EXPORT_SYMBOL(pci_control_last_seq_num_rcvd);
+
+int pci_belongs_to_local_flow(const struct pci * pci)
```

Comment:
This function is used in just one place, and so it is unneded in my opinion. It makes it harder to read the code (you have to remember what are the meanings of -1, 0 and 1. Can't you just do the check 
````
pci_source(pci) == pci_destination(pci)
````
in the only place where the function is used?